packages/components/src/circular-option-picker/circular-option-picker-actions.tsx

```diff
@@ -47,6 +47,7 @@ export function ButtonAction( {
 }: WordPressComponentProps< ButtonAsButtonProps, 'button', false > ) {
 	return (
 		<Button
+			__next40pxDefaultSize
```

Comment:
The other Button instance in this file doesn't need the prop because it is the `link` variant.